qa-tests-backend/src/main/groovy/objects/Deployment.groovy

```diff
@@ -66,9 +66,9 @@ class Deployment {
     }
 
     static final List<String> TEST_IMAGES_TO_IGNORE_FOR_RATE_LIMIT_CHECK = [
-            "busybox:latest",
```

Comment:
Why remove`busybox:latest` and `nginx:latest` here ? Do the tests hit rate limit checks on quay.io?

Reply:
Specifying just `busybox:latest` and `nginx:latest` would result in pulling the images from `docker.io` by default, which results in pull rate limit error sometimes. Hence explicitly specifying the full image from `quay.io`.

Reply:
The list of images `TEST_IMAGES_TO_IGNORE_FOR_RATE_LIMIT_CHECK` is not used for pulling, it is used when `setImage` is called. Removing `busybox:latest` and `nginx:latest` here changes behavior of that warning and I don't see the need for adding `quay.io`.

Reply:
changes reverted